python/lsst/ts/standardscripts/utils.py

```diff
@@ -92,3 +93,24 @@ def format_as_list(value, recurrences):
     value_as_list = [value] * recurrences
 
     return value_as_list
+
+
+def get_topic_time_utc(topic):
+    """
+    Reformat a topic command time from TAI unix to UTC.
```

Comment:
This should be in the previous line, e.g.:

```
"""Reformat a topic command time from TAI unix to UTC.
```

Reply:
Change adopted.